packages/victory-core/src/index.d.ts

```diff
@@ -88,6 +88,7 @@ export interface CallbackArgs {
   active: boolean;
   datum: any;
   horizontal: boolean;
+  index: number;
```

Comment:
actually I think this needs to be `number | string`

Reply:
I know it ends up as a string in the dom, but don't we always want the input to be a number?

Reply:
This prop is set by components like `VictoryArea`. In the case where they only render a single component for the entire dataset they set the value "all"

Reply:
aaah, yeah thats right, fixed 👍